llvm/lib/Target/Mips/MipsISelLowering.cpp

```diff
@@ -3626,33 +3626,16 @@ static SDValue createStoreLR(unsigned Opc, SelectionDAG &DAG, StoreSDNode *SD,
 
 // Expand an unaligned 32 or 64-bit integer store node.
 static SDValue lowerUnalignedIntStore(StoreSDNode *SD, SelectionDAG &DAG,
-                                      bool IsLittle, bool IsCHERI) {
+                                      bool IsLittle,
+                                      const MipsTargetLowering &TL) {
   SDValue Value = SD->getValue(), Chain = SD->getChain();
   EVT VT = Value.getValueType();
   SDValue BasePtr = SD->getBasePtr();
 
   // CHERI doesn't have a capability version of SDR / SDL, so we turn this into
   // a sequence of byte stores and shifts.
```

Comment:
This comment should be updated to reflect the fact that we use generic target-independent logic.